providers/standard/src/airflow/providers/standard/sensors/time.py

```diff
@@ -116,8 +116,10 @@ def execute(self, context: Context) -> None:
                 ),
                 method_name="execute_complete",
             )
+        else:
+            super().execute(context)
 
-    def execute_complete(self, context: Context) -> None:
+    def execute_complete(self, context: Context, event: Any = None) -> None:
         return
```

Comment:
```suggestion
        return None
```

This is more explicit.